core/tests/protractor_utils/ExplorationEditorSettingsTab.js

```diff
@@ -161,25 +169,27 @@ var ExplorationEditorSettingsTab = function() {
   };
 
   this.expectCategoryToBe = async function(category) {
+    var containerLocator = by.css('.e2e-test-exploration-category-dropdown');
     await waitFor.presenceOf(
-      explorationCategoryInput,
+      explorationCategory,
       'Exploration category input takes too long to be visible.');
-    expect(await explorationCategoryInput.$('option:checked').getText()).
+    expect(await element(containerLocator).getText()).
       toEqual(category);
   };
 
   this.expectFirstStateToBe = async function(firstState) {
     await waitFor.presenceOf(
       initialStateSelect, 'Initial state select takes too long to be visible.');
-    expect(await initialStateSelect.$('option:checked').getText()).
+    expect(await initialStateSelect.getText()).
       toEqual(firstState);
   };
 
   this.expectLanguageToBe = async function(language) {
     await waitFor.presenceOf(
       explorationLanguageInput, 'Language input takes too long to be visible.');
-    expect(await explorationLanguageInput.$('option:checked').getText()).
+    expect(await explorationLanguageInput.getText()).
       toEqual(language);
+    await action.click('Neutral element', neutralElement);
```

Comment:
Why is this added? Neutral element clicks can cause hard-to-debug issues, so we should have a good reason for adding them

Reply:
removed